ls1intum/Artemis
# Exam Exercises Overview Page

### Checklist

### Motivation and Context


Currently students do not have an overview over the exercises during an exam. In this PR we change this. 
### Description

We introduce a new Exam Exercise Overview Board which appears at the start of the exam. Students are able to see an an overview over all exercises. The View is integrated into the currently used page layout. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> participate in an exam. 
3. Test if the exam overview displays all exercises correctly. 
4. Check that navigation works. 
5. Check that login / logout during an exam works. 

### Test Coverage



ExamDetailComponent: 100%
ExamNavigationBar: 98%
ExamParticipationservice: 87%
ExamExerciseOverviewPage: 90%
### Screenshots


<img width="1418" alt="Bildschirmfoto 2021-06-29 um 16 14 01" src="https://user-images.githubusercontent.com/42205689/123813280-0d7fd280-d8f5-11eb-959a-f78661817654.png">
